src/cljs/weasel/impls/websocket.cljs

```diff
@@ -20,7 +20,7 @@
   ([auto-reconnect?]
      (websocket-connection auto-reconnect? nil))
   ([auto-reconnect? next-reconnect-fn]
-     (goog.net.WebSocket. auto-reconnect? next-reconnect-fn)))
+     (goog.net.WebSocket. (or auto-reconnect? false) (or next-reconnect-fn false))))
```

Comment:
Seems to me those can be some let-bindings instead, as the code looks a bit weird as proposed.

Reply:
yep, and `false` doesn't seem to be valid according to https://google.github.io/closure-library/api/goog.net.WebSocket.Options.html

maybe
```
(goog.net.WebSocket. (clj->js
                      (cond-> {}
                        auto-reconnect?   (assoc "autoReconnect" auto-reconnect?)
                        next-reconnect-fn (assoc "getNextReconnect" next-reconnect-fn))))
```

or, to make things clearer, using something like `assoc-some` from https://github.com/weavejester/medley/blob/1.3.0/src/medley/core.cljc#L38